roles/repository/tasks/share.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Initialize accumulator for dynamic share properties
+  ansible.builtin.set_fact:
+    dynamic_share_properties: {}
+
+- name: Configure identity service properties when available
+  when: repository_identity_url | length > 0
```

Comment:
I've seen we've spreaded that syntax in many different places lately (and not so lately).
Do we want make it a coding standard? I personnaly prefer the shorthand `repository_identity_url` which evaluate the same way as far as I can tell

Reply:
not sure, because in the end with `repository_identity_url: ''` still evaluates to false.

The other similar usages I see are related to secret where we could enforce a minimum length rather than plain `> 0`